subscriber/src/subscriber/save.bal

```diff
@@ -87,26 +95,35 @@ function saveImagePdf(map<json> imageJson) {
     }
 }
 
-# Return the file name to store this result using the format:
-# 	NNN-{TypeCode}-{LevelCode}[--{EDCode[--{PDCode}]].{ext}
+# Return the file name to store this result using the election format:
+# 	NNN-{TypeCode}-{LevelCode}[-{Code}[--{EDName[--{PDName}]]].{ext}
 # where
 # 	NNN			Sequence number of the result with 0s if needed (001, 002, ..).
-#	{TypeCode}	Result type- first preference or 2nd/3rd preference. “PE1” for
+#	{TypeCode}	Presidential result type- first preference or 2nd/3rd preference. “PE1” for
 #				first preference count and “PE2” for 2nd/3rd preference counts.
-#	{LevelCode}	Result level: PD for polling division, ED for electoral district,
+#               Parliamentary Result type- "R_V", "R_VI", "R_S", "R_SI", "R_SC", "R_VSN", "R_NC".
+#	{LevelCode}	Presidential result level: PD for polling division, ED for electoral district,
 #				NI for national incremental result and NF for national final result.
+#               Parliamentary result level: PD for polling division, ED for electoral district, and N for national.
+#   {Code}      If ED results, then 2 digit code of the district. If PD results then 2 digit ED code followed by one
```

Comment:
Indentation issue? Let's fix separately.

Reply:
Seems a bug in git. Try editing the file using git UI